src/Python/wcon/wcon_schema.json

```diff
@@ -1,164 +0,0 @@
-{
-    "$schema": "http://json-schema.org/draft-04/schema#",
-    "title": "Worm tracker Commons Object Notation (WCON)",
```

Comment:
See #86: basically `wcon_schema.json` was having to be stored in two places in the repo, in the root folder and also at `src/Python/wcon/`, but I discovered that the packager setup file `setup.py` lets you specify files outside the package folder to be packaged alongside during the packaging process.  So we no longer need to have two different identical files `wcon_schema.json`.  It's still stored in the root folder.